doc/source/users/taurus/experimentconfiguration.rst

```diff
@@ -102,25 +102,18 @@ a given channel or its controller:
 
 .. _expconf_ui_showplots:
 
-Show / Hide current scan plot(s)
+View current scan plot(s)
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-The Experiment Configuration widget provides a button to show/hide plots of the
-current scan. When this button is checked, the values of `plot type` and
-`plot axes` :ref:`in the channel configuration <expconf_ui_measurementgroup_channel>`
-determine how many plot widgets will be spawned to show the channels involved
-in a scan when the scan is run.
+Plots need to be configured previously as explained
+:ref:`in the channel configuration <expconf_ui_measurementgroup_channel>`
+once the values of plot type and plot axes are set, and storage path is defined.
```

Comment:
I would just put "(plot type and plot axes)" after the link and finish the sentence here.